src/Microsoft.TestPlatform.PlatformAbstractions/netstandard1.0/Process/ProcessHelper.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Helpers
```

Comment:
Not sure if there should be any `Helper` in the namespace. Process abstraction directly belongs to PlatformAbstractions since it is a platform specific concept?